src/main/app/dashboard/navbar/navbar.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="nav-bg">
+  <div class="blur">
+    <nav class="navbar navbar-expand-lg navbar-dark px-0 py-3">
+      <div class="container">
+        <a class="navbar-brand" routerLink="login">
+          <img
+            src="https://i.ibb.co/jHjLYmM/logo-white.png"
+            class="h-8"
+            width="160" />
+        </a>
+
+        <div class="navbar-nav">
+          <img
+            src="https://www.pngall.com/wp-content/uploads/5/User-Profile-PNG-Image.png"
+            width="50"
+            class="user-img" />
+          <div class="mt-1">
+            <a class="nav-item nav-link d-block p-0">Haris Khan</a>
```

Comment:
The assumption, this will be updated in the upcoming sprint.
Also, never use your own name instead use generic test name e.g. **John Doe**

Reply:
changed